osf/migrations/0002_cross_app_migrations.py

```diff
@@ -0,0 +1,49 @@
+# Generated by Django 3.2 on 2022-08-08 15:38
+
+from django.db import migrations, models
+import django.db.models.deletion
+from django.db.utils import ProgrammingError
+
+
+class AddFieldIfNotExists(migrations.AddField):
```

Comment:
on Production these should already exist hence the error catching, locally this is necessary due to the app structure, potentially there are better ways of doing this.